requirements.testing.txt

```diff
@@ -2,6 +2,7 @@
 pre-commit>=2.2.0
 
 # pre-commit tasks in Makefile need these
+anyio==3.5.0
```

Comment:
F34 has 2.0.2 and F35 has 3.2; this is perhaps too new. I would suggest seeing if F34 version works as start.